### PR TITLE
add FTX_US exchange (issue #244)

### PR DESCRIPTION
**Pull request by AlgoTrader5**
add FTX_US exchange

### CODE REVIEW

File: cryptofeed/exchange/ftx_us.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+LOG = logging.getLogger('feedhandler')
+
+
+class FTXUS(FTX):
+    id = FTX_US
+
+    def __init__(self, pairs=None, channels=None, callbacks=None, **kwargs):
+        super().__init__(pairs=pairs, channels=channels, callbacks=callbacks, **kwargs)
+        self.ws_endpoint = 'wss://ftx.us/ws/'
+        self.rest_endpoint = 'https://ftx.us/api'
```
Comment (bmoscon) [inline cryptofeed/exchange/ftx_us.py:22]: what is this for? it doesnt seem to be used anywhere for FTX

File: examples/demo_ftx_us.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+Please see the LICENSE file for the terms and conditions
+associated with this software.
+'''
+from decimal import Decimal
+
+from cryptofeed.callback import TickerCallback, TradeCallback, BookCallback, FundingCallback
+from cryptofeed import FeedHandler
+from cryptofeed.exchanges import FTXUS, FTX
+from cryptofeed.defines import L2_BOOK, BID, ASK, TRADES, TICKER, FUNDING, COINBASE, OPEN_INTEREST, \
```
Comment (bmoscon) [inline examples/demo_ftx_us.py:12]: you arent using all these imports, please clean up

File: examples/demo_ftx_us.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+async def funding(**kwargs):
+    print(f"Funding Update for {kwargs['feed']}")
+    print(kwargs)
+
+
+async def oi(feed, pair, open_interest, timestamp, receipt_timestamp):
+    print(f'Timestamp: {timestamp} Feed: {feed} Pair: {pair} open interest: {open_interest}')
+
+
+async def volume(**kwargs):
```
Comment (bmoscon) [inline examples/demo_ftx_us.py:46]: not used

File: examples/demo_ftx_us.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+async def book(feed, pair, book, timestamp, receipt_timestamp):
+    print(f'Timestamp: {timestamp} Feed: {feed} Pair: {pair} Book Bid Size is {len(book[BID])} Ask Size is {len(book[ASK])}')
+
+
+async def funding(**kwargs):
+    print(f"Funding Update for {kwargs['feed']}")
+    print(kwargs)
+
+
+async def oi(feed, pair, open_interest, timestamp, receipt_timestamp):
```
Comment (bmoscon) [inline examples/demo_ftx_us.py:42]: not used

File: examples/demo_ftx_us.py [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    assert isinstance(amount, Decimal)
+    assert isinstance(price, Decimal)
+    print(f"Timestamp: {timestamp} Cryptofeed Receipt: {receipt_timestamp} Feed: {feed} Pair: {pair} ID: {order_id} Side: {side} Amount: {amount} Price: {price}")
+
+
+async def book(feed, pair, book, timestamp, receipt_timestamp):
+    print(f'Timestamp: {timestamp} Feed: {feed} Pair: {pair} Book Bid Size is {len(book[BID])} Ask Size is {len(book[ASK])}')
+
+
+async def funding(**kwargs):
```
Comment (bmoscon) [inline examples/demo_ftx_us.py:37]: not used

File: examples/demo_ftx_us.py [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+async def trade(feed, pair, order_id, timestamp, side, amount, price, receipt_timestamp):
+    assert isinstance(timestamp, float)
+    assert isinstance(side, str)
+    assert isinstance(amount, Decimal)
+    assert isinstance(price, Decimal)
+    print(f"Timestamp: {timestamp} Cryptofeed Receipt: {receipt_timestamp} Feed: {feed} Pair: {pair} ID: {order_id} Side: {side} Amount: {amount} Price: {price}")
+
+
+async def book(feed, pair, book, timestamp, receipt_timestamp):
```
Comment (bmoscon) [inline examples/demo_ftx_us.py:33]: not used (but could be used) - you could add books here in the demo

File: cryptofeed/exchange/ftx_us.py (hunk 1 above)
Comment (AlgoTrader5) [reply cryptofeed/exchange/ftx_us.py]: removed endpoints and added address that's used in FTX class

**Pull request merged**